docs/drf_yasg.rst

```diff
@@ -124,7 +128,11 @@ provides the :py:class:`~drf_spectacular.types.OpenApiTypes` enum:
 - :py:data:`~drf_yasg.openapi.TYPE_STRING` with :py:data:`~drf_yasg.openapi.FORMAT_SLUG` has no direct equivalent. Use
   :py:attr:`~drf_spectacular.types.OpenApiTypes.STR` or :py:class:`str` instead.
 
-- :py:data:`~drf_yasg.openapi.TYPE_ARRAY` has no direct equivalent.
+- :py:data:`~drf_yasg.openapi.TYPE_ARRAY` is handled by setting the
+  :py:attr:`~drf_spectacular.utils.OpenApiParameter.many` attribute to `True`
```

Comment:
the linting complained about the single backticks I believe.

Reply:
Gotcha.  Will fix.